feat(scheduler-targets-alpha): `KinesisDataFirehosePutRecord` Target (#27842)
aws/aws-cdk: packages/@aws-cdk/aws-scheduler-targets-alpha/README.md

  target: new targets.CodeBuildStartBuild(project),
});
```

## Put records to an Amazon Kinesis Data Firehose

> Use the `KinesisDataFirehosePutRecord` target to put records to an Amazon Kinesis Data Firehose.

Reviewer: ```suggestion
Use the `KinesisDataFirehosePutRecord` target to put a record to a Kinesis Data Firehose delivery stream.
```

The code snippet below creates an event rule with a delivery stream as a target
called every hour by Event Bridge Scheduler with a custom payload.

```ts

> import * as kinesisfirehose from 'aws-cdk-lib/aws-kinesisfirehose';
> 
> declare const deliveryStream: kinesisfirehose.CfnDeliveryStream;

Reviewer: ```suggestion
import * as firehose from 'aws-cdk-lib/aws-kinesisfirehose';

declare const deliveryStream: firehose.CfnDeliveryStream;
```
